### PR TITLE
Update remindme.js

### PR DESCRIPTION
**Pull request by Soumil07**
Adds additional utilities to remindme

### CODE REVIEW

File: commands/Tools/remindme.js [hunk 1]
```diff
@@ -1,16 +1,46 @@
...
 			usageDelim: ', '
 		});
 	}
 
 	async run(msg, [when, ...text]) {
+		if (msg.flags.list) {
+			const reminders = this.client.schedule.tasks.filter(task => task.taskName === 'reminder' && task.data.user === msg.author.id);
+			if (reminders.length === 0) throw 'You have no pending reminders currently.';
+
+			const send = [];
```
Comment (kyranet) [inline commands/Tools/remindme.js]: Could be replaced with:

```javascript
const send = reminders.map((reminder, i) =>
	`[${i + 1}] ${reminder.id} :: ${reminder.data.text || 'No text'}`);
```

File: commands/Tools/remindme.js [hunk 2]
```diff
@@ -1,16 +1,46 @@
...
+			for (let i = 0; i < reminders.length; i++) {
+				send.push(`[${i + 1}] ${reminders[i].id} :: ${reminders[i].data.text || 'No text'}`);
+			}
+
+			return msg.send(`**Your reminders**: ${codeBlock('asciidoc', send.join('\n'))}`);
+		}
+
+		if (msg.flags.delete) {
+			return this.client.schedule.delete(msg.flags.delete)
```
Comment (kyranet) [inline commands/Tools/remindme.js]: This implies that if user 'x' creates a reminder, the user 'y' can remove it aswell. Please do a check before.

File: commands/Tools/remindme.js [hunk 3]
```diff
@@ -1,16 +1,46 @@
...
+				.then(() => msg.send('Succesfully removed reminder.'))
+				.catch(() => msg.send('This reminder ID does not exist.'));
+		}
+
+		if (msg.flags.clear) {
+			const promises = [];
+			const reminders = this.client.schedule.tasks.filter(task => task.taskName === 'reminder' && task.data.user === msg.author.id);
+			if (reminders.length === 0) throw 'You have no reminders.';
+			for (const reminder of reminders) promises.push(this.client.schedule.delete(reminder.id));
```
Comment (kyranet) [inline commands/Tools/remindme.js]:
```javascript
await Promise.all(reminders.map(reminder => reminder.delete()));
```

File: commands/Tools/remindme.js [hunk 4]
```diff
@@ -1,16 +1,46 @@
...
+
+			return Promise.all(promises)
```
Comment (kyranet) [inline commands/Tools/remindme.js]: Don't make a Promise.all and with a then return the message, use `await` instead to keep the consistency.

File: commands/Tools/remindme.js [hunk 5]
```diff
@@ -1,16 +1,46 @@
...
+				.then(() => msg.send(`Deleted ${reminders.length} reminders. ðŸ‘Œ`));
```
Comment (kyranet) [inline commands/Tools/remindme.js]: What is that weird unicode, `ðŸ‘Œ`?

Comment (T4LO5) [reply commands/Tools/remindme.js]: screen shot 2018-01-13 at 10 36 40 am
That corresponds to 👌

File: commands/Tools/remindme.js [hunk 6]
```diff
@@ -1,16 +1,45 @@
-const { Command } = require('klasa');
+const { Command, util: { codeBlock } } = require('klasa');
 
 module.exports = class extends Command {
 
 	constructor(...args) {
 		super(...args, {
 			description: 'creates a reminder',
```
Comment (MrJacz) [inline commands/Tools/remindme.js]: Add more descriptive description for command flags.

**Comment by kyranet** [comment]
Any status in this PR?

**Comment by kyranet** [comment]
Closed this PR as there was no update in over a month.

**Pull request closed without merging**